src/deadline/job_attachments/asset_manifests/_canonical_json.py

```diff
@@ -15,7 +15,9 @@ def canonical_path_comparator(path: BaseManifestPath):
     """
     # Sort by UTF-16 values as per the spec
     # https://www.rfc-editor.org/rfc/rfc8785.html#name-sorting-of-object-propertie
-    return path.path.encode("utf-16_be")
+    # Use the "surrogatepass" error handler because filenames encountered in the wild
+    # include surrogates.
+    return path.path.encode("utf-16_be", errors="surrogatepass")
```

Comment:
And I thought this was some magic keyword :)  
https://docs.python.org/3/library/codecs.html